tests/sles4sap/sapconf.pm

```diff
@@ -83,28 +83,36 @@ sub run_developers_tests {
     type_string "cd\n";
 }
 
+sub verify_sapconf_service {
+    my $svc  = shift;
+    my $desc = shift;
```

Comment:
It's more common in os-autoinst-distri-opensuse to use array-assignment in one line to all parameters. Would you mind changing that?

Reply:
Sure, no problem.